src/main/java/seedu/address/model/person/Description.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's description in the address book.
+ * Guarantees: immutable;
+ */
+public class Description {
+    public static final String MESSAGE_CONSTRAINTS = "Description can take on any value or just blank,"
```

Comment:
I'm not sure whether we should allow blank description i.e. `add ... d/` since address (which is optional as well) does not allow that

Reply:
Yeah, that's the original implementation, but in that case there's no way for me to allow user to remove description of contact using `d/` in edit command, because of the reason explained above, there's no way for me to pass the new edit info outside because to change the description back to `Optional.empty()` I have to pass in a null but `editPersonDescriptor` must have at least one non-null field else the program takes it as not changing anything. Address does not have these concerns because there's no way to remove the address of a person